src/invocation/ClusterService.ts

```diff
@@ -45,13 +44,20 @@ class MemberListSnapshot {
     constructor(
         public version: number,
         public readonly members: Map<string, MemberImpl>,
-        public readonly memberList: MemberImpl[]
+        public readonly memberList: MemberImpl[],
+        public readonly clusterUuid: UUID
     ) {}
 }
 
-const EMPTY_SNAPSHOT = new MemberListSnapshot(-1, new Map<string, MemberImpl>(), []);
+/**
+ * Initial list version is used at the start and also after cluster has changed with blue-green deployment feature.
+ * In both cases, we need to fire InitialMembershipEvent.
+ */
+const INITIAL_MEMBER_LIST_VERSION = -1;
+const EMPTY_SNAPSHOT = new MemberListSnapshot(INITIAL_MEMBER_LIST_VERSION, new Map<string, MemberImpl>(), [], null);
```

Comment:
This is only used to initialize `memberListSnapshot`. Maybe we can get rid of the variable. WDYT?

Reply:
Absolutely you are right. I could not see this when rearrange the code. Fixed.